hw/ip/edn/data/edn.hjson

```diff
@@ -0,0 +1,264 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "edn",
+  clock_primary: "clk_i",
+  bus_device: "tlul",
+  bus_host: "none",
+  param_list: [
+    { name: "NumEndPoints",
+      desc: "Number of end points to be supported by this block",
+      type: "int",
+      default: "4",
+      local: "true"
+    },
+    { name: "EndPointBusWidth0",
+      desc: "Power of two width of the obsfucated entropy data bus for end point 0",
+      type: "int",
+      default: "64",
+      local: "true"
+    },
+    { name: "EndPointBusWidth1",
+      desc: "Power of two width of the obsfucated entropy data bus for end point 1",
+      type: "int",
+      default: "32",
+      local: "true"
+    },
+    { name: "EndPointBusWidth2",
+      desc: "Power of two width of the obsfucated entropy data bus for end point 2",
+      type: "int",
+      default: "16",
+      local: "true"
+    },
+    { name: "EndPointBusWidth3",
+      desc: "Power of two width of the obsfucated entropy data bus for end point 3",
+      type: "int",
+      default: "8",
+      local: "true"
+    },
+  ],
+  interrupt_list: [
+    { name: "edn_cmd_req_done"
+      desc: "Asserted when a software csrng request has completed."}
+    { name: "edn_fifo_err"
+      desc: "Asserted when a FIFO error occurs."}
+  ],
+  inter_signal_list: [
+    { struct:  "edn",  // edn_req_t, edn_rsp_t
+      type:    "req_rsp",
+      name:    "edn_cmd",
+      act:     "rsp",
+      width:   "4",
+      package: "edn_pkg",
+    }
+  ],
+
+  regwidth: "32",
+  registers: [
+    { name: "REGEN",
+      desc: "Register write enable for all control registers",
+      swaccess: "rw1c",
+      hwaccess: "hro",
+      fields: [
+        {
+            bits:   "0",
+            desc: ''' When true, the EDN_CONF, EDN_RCT_HEALTH, EDN_APT_HEALTH and EDN_SEED registers can be modified.
+            When false, they become read-only. Defaults true, write one to clear. Note that this needs to be
+            cleared after initial configuration at boot in order to lock in the listed register settings.
+            '''
+            resval: 1
+        },
+      ]
+    },
+    { name: "CTRL",
+      desc: "EDN control register",
+      swaccess: "rw",
+      hwaccess: "hro",
+      fields: [ 
+        {
+            bits: "0",
+            name: "EDN_ENABLE",
+            desc: '''
+                  Setting this bit will enable the EDS module. 
```

Comment:
I think "EDN" (do you use a Dvorak layout like me, by any chance?) :-)

Reply:
Good catch. Do you really use a Dvorak keyboard? I use a Colemak.

Reply:
Mark, after all these years, I never knew...  ;)

Reply:
Hah! I switched to Dvorak at university when my really poor touch-typing was giving me RSI. Learning a new keyboard layout made it much easier to force myself to learn to touch-type "properly" (no temptation to "just type it already" because I couldn't remember which key was which!). The choice of Dvorak was pretty random: in hindsight, maybe not the best one, but I'm not sure my typing neurons are quite as plastic nowadays :-D

Reply:
Resolved by Mark.  Hopefully I don't have to switch to Dvorak or Colemak to resolve this one.  I have enough trouble with subtle changes like qwertz for German.